### DISCUSSION TITLE
Error ocurred

### DISCUSSION THREAD

**Discussion by sergiosmg96** [PaddlePaddle/PaddleSpatial#224] [BaiduKDDCup2022]
I would like to know the full error log, if possible.
The message is the following: [PendingTimeout] pending timeout(>7200), evict by system.
The name of my team is Innova Team and the submit time is 2022-06-23 22:11. Thank you

**Comment by ethanyxx** [comment]
Hi,

In the current phase, the system sets a maximum 2 hours pending time.  Any process triggers the pending timeout error would be killed and no log would be generated. 

Regards,
